<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>289</width>
    <height>85</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="frameRange_layout">
     <property name="spacing">
      <number>2</number>
     </property>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="rootFrames_layout">
     <property name="spacing">
      <number>2</number>
     </property>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="options_layout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <layout class="QFormLayout" name="options1_layout">
       <property name="horizontalSpacing">
        <number>2</number>
       </property>
       <property name="verticalSpacing">
        <number>2</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="originFrame_label">
         <property name="text">
          <string>Origin Frame</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="originFrame_spinBox">
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::ButtonSymbols::NoButtons</enum>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>9999999</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="sceneScale_label">
         <property name="text">
          <string>Scene Scale</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="sceneScale_doubleSpinBox">
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::ButtonSymbols::NoButtons</enum>
         </property>
         <property name="showGroupSeparator" stdset="0">
          <bool>true</bool>
         </property>
         <property name="decimals">
          <number>3</number>
         </property>
         <property name="minimum">
          <double>0.001000000000000</double>
         </property>
         <property name="maximum">
          <double>99999999.000000000000000</double>
         </property>
         <property name="value">
          <double>10.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="options3_layout">
       <property name="spacing">
        <number>2</number>
       </property>
       <item>
        <widget class="QCheckBox" name="solveFocalLength_checkBox">
         <property name="text">
          <string>Solve Focal Length</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="solveLensDistortion_checkBox">
         <property name="text">
          <string>Solve Lens Distortion</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
